test.js

```
@@ -4,3 +4,49 @@ import m from '.';
 test('empty test', t => {
 	t.is(m.thrown(), false);
 });
+
+test('out test', t => {
```

Rename this to:
'should throw InternalServer error with an invalid type'